apps/explorer/src/components/Activity/TransactionsActivityTable.tsx

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '@mysten/core';
+import { ArrowRight12 } from '@mysten/icons';
+import { useQuery } from '@tanstack/react-query';
+import { useState } from 'react';
+
+import { genTableDataFromTxData } from '../transactions/TxCardUtils';
+
+import { useGetTransactionBlocks } from '~/hooks/useGetTransactionBlocks';
+import { Link } from '~/ui/Link';
+import { Pagination } from '~/ui/Pagination';
+import { PlaceholderTable } from '~/ui/PlaceholderTable';
+import { TableCard } from '~/ui/TableCard';
+import { Text } from '~/ui/Text';
+import { numberSuffix } from '~/utils/numberUtil';
+
+const DEFAULT_TRANSACTIONS_LIMIT = 20;
+
+interface Props {
+    disablePagination?: boolean;
+    refetchInterval?: number;
+    initialLimit?: number;
+}
+
+export function TransactionsActivityTable({
```

Comment:
Is this replacing an old component? If so can we remove that or no?

Reply:
not yet, unfortunately. once the address page transactions can be paginated, it can handle both